SwiftIntro.playground/Pages/Untitled Page.xcplaygroundpage/Contents.swift

```diff
@@ -68,17 +102,23 @@ struct RunApp {
                 isPlaying: false
             )
         ]
+    }()
+    
+    var currentSong: Song {
+        didSet {
+            print(oldValue)
+            print(currentSong)
+        }
     }
 }
 
-var runApp = RunApp(id: "1", startTime: Date())
-//print(runApp.elapsedTime())
+var runApp = RunApp(id: "1", startTime: Date(), currentSong: Song(title: "Tralala", authorName: "Author", isPlaying: true))
```

Comment:
We didn't have enough time for our lecture, but there is nice rule of 3 or more parameters when calling a function. 
Don't you think that this code is more readable?

```suggestion
var runApp = RunApp(
    id: "1",
    startTime: Date(),
    currentSong:
        Song(
            title: "Tralala",
            authorName: "Author",
            isPlaying: true
        )
)
```